workflows/templates/initialize-schema.yaml

```diff
@@ -0,0 +1,58 @@
+---
```

Comment:
the templates need to de deployed via https://github.com/thoth-station/thoth-application/blob/master/init-job/base/kustomization.yaml

Reply:
I will open PR there